JSDemos/Demos/Form/GroupedFields/description.md

```diff
@@ -1 +1,5 @@
-This demo illustrates how to&nbsp;organize Form items into groups and tabbed pages. For this purpose, an&nbsp;array containing &laquo;group&raquo; and &laquo;tabbed&raquo; items is&nbsp;passed to&nbsp;the items property.
\ No newline at end of file
+You can organize Form items in groups and tabs. To do this, declare items of the [Group](/Documentation/ApiReference/UI_Components/dxForm/Item_Types/GroupItem/) and [Tabbed](/Documentation/ApiReference/UI_Components/dxForm/Item_Types/TabbedItem/) types in the [items[]](/Documentation/ApiReference/UI_Components/dxForm/Configuration/#items) array. Each group and tab can have their own layout and contain any other item types.
+
+To create a group item, assign "group" to the [itemType](/Documentation/ApiReference/UI_Components/dxForm/Item_Types/GroupItem/#itemType) property. To specify a group's title, use the [caption](/Documentation/ApiReference/UI_Components/dxForm/Item_Types/GroupItem/#caption) property. This demo shows three groups. The `Personal Data` group is nested in the `System Information` group.  
+
+The Form uses the [TabPanel](/Documentation/ApiReference/UI_Components/dxTabPanel/) component to display tabs. To create a tabbed item, assign *"tabbed"* to the [itemType](/Documentation/ApiReference/UI_Components/dxForm/Item_Types/TabbedItem/#itemType) property. This demo shows a tabbed item nested in the `Contact Information` group. You can specify the tab panel's settings in the [tabPanelOptions](/Documentation/ApiReference/UI_Components/dxForm/Item_Types/TabbedItem/#tabPanelOptions) object. This demo sets the [deferRendering](/Documentation/ApiReference/UI_Components/dxTabPanel/Configuration/#deferRendering) property to **false**.
```

Comment:
I moved the first sentence closer to the information about tab panel settings - it seems more straightforward.

```suggestion
To create a tabbed item, assign *"tabbed"* to the [itemType](/Documentation/ApiReference/UI_Components/dxForm/Item_Types/TabbedItem/#itemType) property. This demo shows a tabbed item nested in the `Contact Information` group. The Form uses the [TabPanel](/Documentation/ApiReference/UI_Components/dxTabPanel/) component to display tabs. You can specify the tab panel's settings in the [tabPanelOptions](/Documentation/ApiReference/UI_Components/dxForm/Item_Types/TabbedItem/#tabPanelOptions) object. This demo sets the [deferRendering](/Documentation/ApiReference/UI_Components/dxTabPanel/Configuration/#deferRendering) property to **false**.
```